docs/table_design/Data_distribution.md

```diff
@@ -151,6 +159,44 @@ AGGREGATE KEY(site_id, city_code, user_name)
 DISTRIBUTED BY HASH(site_id,city_code);
 ```
 
+#### Random bucketing (since v3.1)
+
+For data in a partition, StarRocks distributes the data randomly across all buckets, which is not based on specific column values. Additionally, there is no need to set bucketing columns during table creation, simplifying the CREATE TABLE statement. It is suitable for scenarios with relatively small data sizes and requirement for high query performance.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/178f1d0c-d35d-4cd2-889c-cbe7c89fa96d)
I think the wording is a bit ambiguous, as the Chinese says "low requirements for query performance”